src/base/dropdown-element.ts

```diff
@@ -173,16 +173,13 @@ export class DropdownElement extends SgdsElement {
     });
   }
   _getMenuItems(): SgdsDropdownItem[] {
```

Comment:
@clukhei I re-did __getMenuItems() because previously it wasn't obtaining the sgds-dropdown-items in ComboBox correctly; in the old code the `if` statement (`this.shadowRoot.querySelector("slot")`) was run for ComboBox instead because there was a `slot` item for the `form-control-icon` in ComboBox.

Reply:
Its safer to query the general child of the slots then to query a specific name like sgds-dropdown-item. 
you can add an id to <slot id="default"></slot> to make querying more specific to the default slot now 

and query this.shadowRoot.querySelector("slot#default")

Reply:
esp when we allow users to register scoped element names themselves, they might rename sgds-dropdown-item to something else and we cant really control, then this will break.

Reply:
alright, I have updated the implementation